submissions/onyemeMichael/to-do-app/src/components/Todo.js

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from 'react';
+import TodoForm from './TodoForm';
+import { RiCloseCircleLine } from 'react-icons/ri';
+import { TiEdit } from 'react-icons/ti';
+
+
+
+const Todo = ({ todos, completeTodo, removeTodo, updateTodo}) => {
+  const [edit, setEdit] = useState({
+    id: null,
+    value: ''
+  });
+
+  const submitUpdate = value => {
+    updateTodo(edit.id, value);
+    setEdit({
+      id: null,
+      value: ''
+    });
+  };
+
+  if (edit.id) {
+    return <TodoForm edit={edit} onSubmit={submitUpdate} />;
+  };
+
+ console.log(todos);
+  return todos.map((todo, index) => (
+    <div
+      className={todo.isComplete ? 'todo-row complete' : 'todo-row'}
+      key={index}
+    >
+      <div key={todo.id} onClick={() => completeTodo(todo.id)} style={{ textDecoration: todo.isCompleted ? "line-through" : "",color: todo.isCompleted ? "" : ""}}>
```

Comment:
I think you should remove this check `color: todo.isCompleted ? "" : ""` you are not using it.